File: database/bao-web-app/src/main/java/bao/webapp/model/Business.java
package bao.webapp.model;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity(name = "Business")
public class Business {
	
	@Id
	@GeneratedValue
	private long BusinessID;
	
	@Column(name = "BusinessName")
	private String businessName;
	
	@Column(name = "CreateDate")
	private Date createDate;
	
	public long getBusinessID() {
		return BusinessID;
	}
	
	public void setBusinessID(Long businessID) {
		this.BusinessID = businessID;
	}
	
	public String getBusinessName() {
		return businessName;
	}
	
	public void setBusinessName(String businessName) {
		this.businessName = businessName;
	}
	
	public Date getCreateDate() {
		return createDate;
	}
	
	public void setCreateDate(Date createDate) {
		this.createDate = createDate;
	}
}

File: database/src/main/kotlin/com/ossp/database/model/Business.kt
package com.ossp.database.model

import org.hibernate.annotations.GenericGenerator
import java.util.*
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity(name = "Business")
class Business {

    @Id
    @GeneratedValue(generator = "increment")
    @GenericGenerator(name = "increment", strategy = "increment")
    @Column(name = "businessID")
    var businessID: Long = 0

    @Column(name = "businessName")
    var businessName: String? = null

    @Column(name = "createdDate")
    var createDate: Date? = null
}